tests/test_custom_type_setup.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+import gc
+import weakref
+
+import pytest
+
+from pybind11_tests import custom_type_setup as m
+
+
+@pytest.fixture
+def gc_tester():
+    """Tests that an object is garbage collected."""
+
+    weak_refs = []
+
+    def add_ref(obj: Any) -> None:
+        # PyPy does not support `gc.is_tracked`.
+        if hasattr(gc, "is_tracked"):
+            assert gc.is_tracked(obj)
+        weak_refs.append(weakref.ref(obj))
+
+    yield add_ref
+
+    gc.collect()
+    if hasattr(gc, "collect_step"):
+        # PyPy may require additional encouragement to actually collect the
+        # garbage.
+        for i in range(100):
```

Comment:
```suggestion
        for _ in range(100):
```
will stop the linter from complaining